### DISCUSSION TITLE
Let user add/modify PDF/EPUB metadata

### DISCUSSION THREAD

**Discussion by Shohreh** [sumatrapdfreader/sumatrapdf#4633] [Ideas]
Hello,

I often need to add/modify metadata in PDF/EPUB files.

Through CTRL+D, SumatraPDF _does_ display those infos, but doesn't let the user add/modify them.

Cheers

![image](https://github.com/user-attachments/assets/3f1d4c88-7f94-452b-81c2-910cf65438ee)

**Comment by GitHubRulesOK** [comment]
Those are imbedded metadata.
Thus ANY file needs deconstruct to **edit and save** as a different name,
This is the same for a document, an image/picture, a folder, an ePub, or a PDF.

SumatraPDF (unlike Acrobat) avoids becoming a complex editor except for additions to PDF (which do not change the core metadata)

A meta editor like EXIFTool does NOT change PDF metadata (read its description of PDF changes).
> All metadata edits are reversible. While this would normally be considered an advantage, it is a potential security problem 

Complex docX and Acrobat will only change some parts of meta data and may retain some for tracking modifications.
